lib/geo.js

```diff
@@ -0,0 +1,95 @@
+var request = require('request');
+var _ = require('lodash');
+
+// REST url for geocoding api
+var GEOCODE_ENDPOINT = 'http://www.austintexas.gov/GIS/REST/Geocode/COA_Composite_Locator/GeocodeServer/findAddressCandidates';
+
+// local historical districts
+var HIST_DIST_QUERY = 'http://services.arcgis.com/0L95CJ0VTaxqcmED/ArcGIS/rest/services/PLANNINGCADASTRE_zoning_ordinance/FeatureServer/0/query';
+// Returns top N address for given address, ranked by score 0-100
+// could also be done with a cutoff on scores
+var RETURN_N_CANDIDATES = 10;
+
+function getAddressCandidates(addr, callback) {
+    var params = {
+        uri: GEOCODE_ENDPOINT,
+        qs: {
+            street: addr,
+            f: 'pjson'
+        }
+    };
+
+    request(params, function(err, res, body) {
+        var result = null;
+        if(!err && res.statusCode == 200) {
+            body = JSON.parse(body);
+            var top = _.take(body.candidates, RETURN_N_CANDIDATES);
```

Comment:
Hey since we're only using `_.take` we could probably just use `body.candidates.slice(0, RETURN_N_CANDIDATES)` instead of bringing in the entire lodash lib